src/__test__/components/Header/ProjectInformations.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import ProjectInformations from '@components/Header/ProjectInformations';
+import { BrowserRouter } from 'react-router-dom';
+import Header from '@components/Header';
+import '@testing-library/jest-dom/extend-expect';
+
+describe('Project Informations', () => {
+  test('Test title name',  () => {
+    const { getByText } = render(
+      <BrowserRouter>
+        <ProjectInformations setOpen={jest.fn()}/>
+      </BrowserRouter>
+    );
+    const title = getByText('Atlas de Oportunidades');
+    expect(title).toBeTruthy;
+  });
+
+  test('Test to close button behavior', async () => {
+    const { getByRole } = render( 
+      <BrowserRouter>
+        <Header isComparisonModeOn comparisonType="grid" setComparisonType={() => {}} />
+        <ProjectInformations setOpen={jest.fn()}/>
+      </BrowserRouter>
+    );
+    const MenuButton = await getByRole('MenuButton');
```

Comment:
the `getByRole` doesn't have any Promise to be solved, so this await is unnecessary. What do you think instead of `getByRole` use the `findByRole` that return a Promise to be solved 

https://testing-library.com/docs/queries/about